coffee_demo_moves:
  - gripper1_2/cup_open_kettle_open
  - arm1_2/kettle_init_cup_grab
  - arm1_2/kettle_grab_cup_in
  - gripper1_2/cup_close_kettle_close
  - arm2/kettle_init
  - arm1/cup_rotate
  - sleep/kettle_pour
  - arm1_2/kettle_pour_cup_place 
  - sleep/kettle_pour
  - arm2/kettle_init
  - sleep/kettle_pour
  - arm2/kettle_pour
  - sleep/kettle_pour 
  - arm2/kettle_init
  - sleep/kettle_pour 
  - gripper1/cup_open
  - arm1_2/kettle_end_raise_cup_place_up 
  - arm1_2/kettle_end_place_dripper_pre_grasp
  - gripper2/kettle_open
  - arm2/kettle_backup
  - gripper1/dripper_open
  - arm1/dripper_grasp
  - gripper1/dripper_close
  - arm1/dripper_grasp_up
  - arm1/dripper_out
  - arm1/dripper_raise
  - arm1/dripper_above_cup
  - arm1/dripper_half_pour
  - arm1/dripper_full_pour
  - arm1/dripper_above_cup
  - arm1/dripper_raise
  - arm1/dripper_out
  - gripper1/dripper_open

arm1_2:
  kettle_init_cup_grab: [2.4837749004364014, -0.35830846428871155, 1.4157726764678955, -6.265229603741318e-05, -1.0730865001678467, -6.211668369360268e-05,0.5592657327651978, -0.6162416934967041, 1.357011079788208, -0.0021113823167979717, -0.693118691444397, -1.6827927827835083]
  kettle_grab_cup_in: [2.4893195629119873, 0.10498017817735672, 1.1299152374267578, -6.891752855153754e-05, -1.1619857549667358, -6.211668369360268e-05, 0.5635260343551636, -0.30106931924819946, 1.1501082181930542, -0.010707276873290539, -0.7779079675674438, -1.5645122528076172]
  #dripper_init_cup_rotate: [-0.603122353553772, -0.35830846428871155, 1.4157726764678955, -6.265229603741318e-05, -1.0730865001678467, -6.211668369360268e-05,0.10254926979541779, -0.5262980461120605, 0.3006244897842407, -0.0016728162299841642, 0.2373582124710083, -1.5715831518173218]
  kettle_pour_cup_place: [-0.7599347233772278, -0.2355099618434906, 1.5582188367843628, 0.031257230788469315, -1.205219030380249, 0.015787990763783455, -0.03714028000831604, -0.39524826407432556, -0.020637664943933487, -0.0019234254723414779, 0.4426124393939972, -0.7213093042373657]
  kettle_end_raise_cup_place_up: [-0.6509385108947754, -1.0128369331359863, 1.0991216897964478, 0.027567008510231972, -0.11300060153007507, -0.0022362007293850183, 2.7062344551086426, -1.0442194938659668, 1.2487854957580566, -0.0011089455801993608, -0.18830673396587372, -1.6105717420578003]
  kettle_end_place_dripper_pre_grasp: [0.28313198685646057, -0.10128369927406311, 1.209991216659546, 0.027629660442471504, -1.0600212812423706, -0.32400062680244446, 2.706792116165161, -0.7465647459030151, 1.4619474411010742, -0.0012593111023306847, -0.7311444282531738, -1.6103646755218506]


arm2:
  kettle_init: [0.10254926979541779, -0.5262980461120605, 0.3006244897842407, -0.0016728162299841642, 0.2373582124710083, -1.5715831518173218]
  kettle_pour: [-0.03714028000831604, -0.39524826407432556, -0.020637664943933487, -0.0019234254723414779, 0.4426124393939972, -0.7213093042373657]
  kettle_end_raise: [2.7062344551086426, -1.0442194938659668, 1.2487854957580566, -0.0011089455801993608, -0.18830673396587372, -1.6105717420578003]
  kettle_end_place: [2.706792116165161, -0.7465647459030151, 1.4619474411010742, -0.0012593111023306847, -0.7311444282531738, -1.6103646755218506]
  kettle_backup: [2.7036030292510986, -1.6773459911346436, 1.4772971868515015, -0.0012843719450756907, 0.2346871942281723, -1.6106131076812744]
  kettle_end_raise: [2.7062344551086426, -1.0442194938659668, 1.2487854957580566, -0.0011089455801993608, -0.18830673396587372, -1.6105717420578003]
  kettle_end_place: [2.706792116165161, -0.7465647459030151, 1.4619474411010742, -0.0012593111023306847, -0.7311444282531738, -1.6103646755218506]
  kettle_backup: [2.7036030292510986, -1.6773459911346436, 1.4772971868515015, -0.0012843719450756907, 0.2346871942281723, -1.6106131076812744]


gripper2:
  kettle_open: [0.5,0.5]
  kettle_close: [1.0, 1.0]

gripper1:
  dripper_open: [0.85,0.85]
  dripper_close: [1.09,1.09]
  cup_open: [0.0,0.0]
  cup_close: [0.56,0.56]

gripper1_2:
  cup_open_kettle_open: [0.0,0.0,0.5,0.5]
  cup_close_kettle_close: [0.56,0.56,1.0,1.0]

sleep:
  kettle_pour: [1]
  dripper_pull: [25]
  dripper_pour: [5]


arm1:
  # cup_init: [2.4837749004364014, -0.35830846428871155, 1.4157726764678955, -6.265229603741318e-05, -1.0730865001678467, -6.211668369360268e-05] 
  # cup_in: [2.4893195629119873, 0.10498017817735672, 1.1299152374267578, -6.891752855153754e-05, -1.1619857549667358, -6.211668369360268e-05]
  cup_rotate: [-0.603122353553772, -0.35830846428871155, 1.4157726764678955, -6.265229603741318e-05, -1.0730865001678467, -6.211668369360268e-05] 
  #cup_place: [-0.5755929350852966, 0.3824859857559204, 1.185901403427124, 0.02890150249004364, -1.62294340133667, 0.0075471773743629456]
  cup_place_up: [-0.6509385108947754, -1.0128369331359863, 1.0991216897964478, 0.027567008510231972, -0.11300060153007507, -0.0022362007293850183]
  dripper_pre_grasp: [0.28313198685646057, -0.10128369927406311, 1.209991216659546, 0.027629660442471504, -1.0600212812423706, -0.32400062680244446]
  dripper_grasp: [0.29576268792152405, 0.1403348743915558, 1.0167840719223022, 0.027379052713513374, -1.1481856107711792, -0.3256053030490875]
  dripper_grasp_up: [0.29576268792152405, 0.1403348743915558, 1.0167840719223022, 0.027379052713513374, -1.18309221077, -0.3256053030490875]
  # dripper_grasp_up: [0.3087129294872284, 0.1049739196896553, 1.0271341800689697, 0.027360256761312485, -1.1480199098587036, -0.3256053030490875]
  dripper_out: [-0.09312010556459427, -0.05747094750404358, 1.1919223070144653, 0.026959281414747238, -1.116754412651062, -0.3256053030490875]
  dripper_raise: [-0.2421887069940567,-0.9977189898490906,0.6663322448730469,-0.0004072398878633976,0.3089994490146637,-0.0005486973677761853]
  dripper_above_cup: [-1.3527193069458008,-0.9121923446655273,0.9235073328018188,-0.01102053839713335,-0.04133865237236023,-0.3367655873298645]
  dripper_half_pour: [-0.956713080406189,-0.9391390681266785,0.8097871541976929,-0.01222972758114338,0.24042263627052307,-1.9916679859161377]
  # dripper_full_pour: [-0.76074128055,-0.8484561443328857,0.648094117641449,-0.011860079132020473,0.33987146615982056,-2.5093586444854736] OLD VALUE
  dripper_full_pour: [-0.6578302979469299,-0.6893255710601807,0.48040550136,-0.01189140509814024,0.49449020624160767,-2.7012579441070557]
# need to adjust
# - cup place (currently too high) DONE
# - kettle grab (currently too deep) DONE
# - kettle pour (fucked) DONE 
# - pot grab (currently too shallow, angle is a little off) DONE
# - pot lift (currently too high)

# need to create
# - pot over cup
# - pot pour
# - pot place